packages/atomic/src/components/search/facets/atomic-facet/atomic-facet.tsx

```diff
@@ -112,9 +114,14 @@ export class AtomicFacet implements InitializableComponent, BaseFacet<Facet> {
   @Prop({reflect: true}) public withSearch = true;
   /**
    * The sort criterion to apply to the returned facet values.
-   * Possible values are 'score', 'alphanumeric', 'alphanumericDescending', 'occurrences', and 'automatic'.
+   * Possible values are 'score', 'alphanumeric', 'occurrences', and 'automatic'.
    */
   @Prop({reflect: true}) public sortCriteria: FacetSortCriterion = 'automatic';
```

Comment:
Whoops! I temporarily considered removing `alphanumericDescending`, but decided against it since we can't have any breaking changes until V3. I'll scrub through this to make sure it's still there in both the `FacetSortCriterion` types as well as the documentation ✅